Remove working_with_data_collections notebook

emmarogge opened the pull request:
Due to recent changes that prevent users from creating a data collection without access to the Workspace Manager API, remove data collections notebook and all references to it for now.

wnojopra reviewed (changes requested):
The working_with_data notebooks has a lot there, lets move that to a separate PR. This PR should just be the removal of the working_with_data_collections notebook.

emmarogge commented:
> The working_with_data notebooks has a lot there, lets move that to a separate PR. This PR should just be the removal of the working_with_data_collections notebook.

I have moved the data notebook to [its own PR](https://github.com/DataBiosphere/terra-axon-examples/pull/47).

emmarogge committed:
Add notebook demonstrating BigQuery use

Add notebook with examples of accessing BigQuery resources in a cloud
environment. Notebook demonstrates best practices for utilizing BigQuery
data in VWB.
